index.rst

```diff
@@ -306,7 +312,7 @@ This example assumes the component of interest is already in the ``STANDBY`` sta
 
     # ATAOS must be in STANDBY state for this to work. All other CSCs will
     # use their default configurations
-    await atcs.enable(configurationOverride={'ATAOS': 'summit_constant_hex.yaml'})
+    await atcs.enable(override={'ATAOS': 'summit_constant_hex.yaml'})
```

Comment:
This code example is out of date. The ATAOS does not provide this as an override file.

Reply:
see comment below